packages/client/src/components/SafeSettings/components/Owners/index.js

```diff
@@ -1,52 +1,36 @@
 import React from "react";
-import { useHistory } from "react-router-dom";
 import { useModalContext } from "contexts";
 import { useClipboard } from "hooks";
 import EditThreshold from "../EditThreshold";
 import RemoveSafeOwner from "./RemoveSafeOwner";
 import AddSafeOwner from "./AddSafeOwner";
-import { formatAddress } from "utils";
 import Svg from "library/Svg";
 
-const Owners = ({ treasury, proposeRemoveSigner }) => {
+const Owners = ({ treasury }) => {
   const modalContext = useModalContext();
-  const history = useHistory();
-  const { safeOwners, address } = treasury;
+  const { safeOwners } = treasury;
   const verifiedSafeOwners = safeOwners.filter((o) => o.verified);
   const ownersAddressClipboard = useClipboard();
 
-  const onRemoveSafeOwnerSubmit = async (ownerToBeRemoved) => {
-    if (ownerToBeRemoved) {
-      await proposeRemoveSigner(formatAddress(ownerToBeRemoved.address));
-    }
-
-    modalContext.closeModal();
-    history.push(`/safe/${address}`);
-  };
-  const openRemoveOwnerModal = (safeOwner) => {
-    modalContext.openModal(
-      <RemoveSafeOwner
-        safeOwner={safeOwner}
-        onCancel={() => modalContext.closeModal()}
-        onSubmit={onRemoveSafeOwnerSubmit}
-      />
-    );
-  };
   const openAddOwnerModal = () => {
     modalContext.openModal(
       <AddSafeOwner
-        onCancel={() => modalContext.closeModal()}
-        onNext={openEditSignatureThresholdModal}
+        treasury={treasury}
         safeOwners={safeOwners}
       />
```

Comment:
This will add the title to the modal, so we can remove the Add Owner from AddSafeOwner.js
```
modalContext.openModal(
      <AddSafeOwner treasury={treasury} safeOwners={safeOwners} />,
      { headerTitle: "Add Owner" }
);
```